docker/docker-compose.dev.yml

```diff
@@ -32,8 +32,8 @@ services:
       args:
         APP_ENV: dev
     depends_on:
-      - broker:
```

Comment:
this `:` at the end was causing error 
```
ERROR: The Compose file './docker-compose.dev.yml' is invalid because:
services.kafka-setup.depends_on contains {"broker": null}, which is an invalid type, it should be a string

```